test-app/app/templates/docs/lazy-loading.md

```diff
@@ -22,11 +22,11 @@ module.exports = function(environment) {
 - Load the scripts when needed:
 
 ```js
-export default Route.extend({
-  phoneInput: service(),
+export default class Route extends Route {
+  @service phoneInput;
 
-  beforeModel() {
-    return this.get('phoneInput').load()
+  async beforeModel() {
+    await this.phoneInput.load();
```

Comment:
Are we missing a `return` here? Maybe this could be:

```suggestion
  async beforeModel() {
    return await this.phoneInput.load();
```

or maybe even better:

```suggestion
  async beforeModel() {
    return this.phoneInput.load();
```

Reply:
No we actually just want to lazy load (import dynamically) everything within the service so the component can use it later internally

Reply:
Gotcha thanks!